cds_wan_point_to_point_node_port/public_cds/cds-nodeport.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: cds-nodeport
+spec:
+  type: NodePort
+  selector:
+    statefulset.kubernetes.io/pod-name: cds-rwt-0
+  ports:
+    - nodePort: 30007 # public port
+      protocol: UDP
+      port: 7400 # internal port
+  externalTrafficPolicy: Local
```

Comment:
I think it would be useful if we can explain a bit about why we need this config (e.g., CDS leverages source IP addresses for hole punching and this configuration is needed not to replace the source IP address that are replaced by NodePort & Load Balancer with the default setting).